README.md

```diff
@@ -1,2 +1,56 @@
-# diagrams
-Utility tool to generate mermaid diagrams for Warehouse.ai
+# `@wrhs/diagrams`
+
+[![Version npm](https://img.shields.io/npm/v/@wrhs/diagrams.svg?style=flat-square)](https://www.npmjs.com/package/diagrams)
```

Comment:
`https://www.npmjs.com/package/diagrams` is that right? should have `@wrhs` in there right?